compiler/src/iree/compiler/Dialect/Flow/Transforms/Passes.cpp

```diff
@@ -84,6 +84,12 @@ static llvm::cl::opt<bool> clEnableLinalgDetensorize(
     llvm::cl::desc("Enable detensorizing linalg ops to operate on primitives"),
     llvm::cl::init(true));
 
+static llvm::cl::opt<bool> clEnableFusionOnMultiUseAndReduction(
+    "iree-flow-enable-fusion-on-multi-use-and-reduction",
```

Comment:
Could we just call this `iree-flow-enable-aggressive-fusion`?

Reply:
Done.